website_sale_wishlist/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========================
+Ecommerce Wishlist
+========================
+
+Let the users save the products they are interested in and may want to buy
```

Comment:
This module extends the functionality of the ecommerce module and introduces a wishlist feature.
The users are able to save the products they are interested in in a wishlist so that they can confirm the purchase later on.